index.html

```diff
@@ -180,14 +180,15 @@
         </li>
         <li>When the `type` attribute of the [link] element of a <a>preload
         link</a> that is already [in a Document] but was previously not
-        obtained due to the `type` attribute specifying an unsupported
+        obtained due to the `type` attribute not specifying a
+        [parsable MIME type] or specifying an [unsupported MIME type] for the
         <a>request destination</a> is set, removed, or changed.
```

Comment:
I think our options are:
* Remove `request destination` here, leave that underspecified, which may or may not create observable differences between browsers.
* Add a `request destination` context to "unsupported MIME type".

I tend towards the latter, but let me know which you think is best.